web/client/examples/viewer/components/MapPanel.jsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2015, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+var React = require('react');
+
+var {Panel, PanelGroup} = require('react-bootstrap');
+var LayerSwitcher = require('../../../components/LayerTree/LayerTree');
+var {Button, Tooltip, OverlayTrigger} = require('react-bootstrap');
+var {Message} = require('../../../components/I18N/I18N');
+var icon = require('../../../components/LayerTree/images/layers.png');
+
+require("./mapPanel.css");
+
+let MapPanel = React.createClass({
```

Comment:
PanelsAccordion is too vague. This can be a start point to show many panels on the viewer. 
It can be ViewerFloatingPanel. What do you think?